assignments/lambda-classes.js

```diff
@@ -1 +1,93 @@
 // CODE here for your Lambda Classes
+
+class Person {
+    constructor(yeet) {
+        this.name = yeet.name;
+        this.age = yeet.age;
+        this.location = yeet.location;
+        this.gender = yeet.gender;
+
+    }
+    speak() {
+        return `Hello my name is ${this.name} , I am from ${this.location}`
+    }
+}
+
+class Instructor extends Person {
+    constructor(instruct) {
+        super(instruct)
+        this.specialty = instruct.specialty;
+        this.favLanguage = instruct.favLanguage;
+        this.catchPhrase = instruct.catchPhrase;
+
+    }
+
+    demo(subject) {
+        return `Today we are learning about ${subject}`
+    }
+
+    grade(student, subject) {
+        return `${student.name} receives a perfect score on ${subject}.`
+    }
+}
+
+class Student extends Person{
+constructor(studentAttr){
+    super(studentAttr)
+    this.previousBackground = studentAttr.previousBackground;
+    this.classname = studentAttr.classname;
+    this.favSubjects = studentAttr.favSubjects;
+}
+    listsSubjects() {
```

Comment:
This method should be `a method that logs out all of the student's favoriteSubjects one by one.`. 

Given an array, which `favSubjects` is, how might you log each element of the array, one element at a time?